internal/directives/schemas/helm-template-config.json

```diff
@@ -17,7 +17,8 @@
     },
     "releaseName": {
       "type": "string",
-      "description": "ReleaseName to use for the rendered manifests."
+      "description": "ReleaseName to use for the rendered manifests.",
+      "minLength": 1
```

Comment:
I think `releaseName` should also be included in `required` fields on line 6. iirc, `minLength` only kicks in after a field is defined. So `null` or `undefined` remain valid currently.

Reply:
Oh, in that case I need to improve some other configurations as well.

Reply:
Can we fix this on and then circle back to others in a follow-up?

Reply:
Fixed in https://github.com/akuity/kargo/pull/2628/commits/d4e765c6e59e5bb9a7bfee6e68ef4967cae99c1a